# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/ipc/RemoteDecoderManagerChild.h
# Commit: c177bcba1c07
# Full Hash: c177bcba1c076d041e3470c8058bb2b4adaa1123
# Author: Dan Glastonbury <dan.glastonbury@gmail.com>
# Date: 2020-10-21 09:51:28
# Regressor Bug: 1595994
# File Overlap Count: 1
# Description:
#   Bug 1595994 - P8: Use RemoteDecoderModule to communicate with GPU and RDD. r=mattwoodrow
#   
#   Depends on D54879
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D54880
# ==============================================================================

diff -r 57607f8b6d8e -r c177bcba1c07 dom/media/ipc/RemoteDecoderManagerChild.h
--- a/dom/media/ipc/RemoteDecoderManagerChild.h	Tue Oct 20 23:26:27 2020 +0000
+++ b/dom/media/ipc/RemoteDecoderManagerChild.h	Tue Oct 20 23:26:30 2020 +0000
@@ -40,7 +40,7 @@
 
   // Can be called from any thread.
   static nsISerialEventTarget* GetManagerThread();
-  static void LaunchRDDProcessIfNeeded();
+  static void LaunchRDDProcessIfNeeded(RemoteDecodeIn aLocation);
 
   // Can be called from any thread, dispatches the request to the IPDL thread
   // internally and will be ignored if the IPDL actor has been destroyed.